packages/office-addin-debugging/src/debugInfo.ts

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as fs from "fs";
+import { parseNumber } from "office-addin-cli";
+import * as os from "os";
+import * as path from "path";
+
+const processIdFile = "office-addin-debugging.json";
+const processIdFilePath = path.join(os.tmpdir(), processIdFile);
+
+export interface IDebugInfo {
+    devServer: IDevServerInfo;
+}
+
+export interface IDevServerInfo {
+    processId: number;
+}
+
+export function getDebuggingInfoPath(): string {
+    return processIdFilePath;
+}
+
+/**
+ * Creates the DebugInfo object
+ */
+function createDebugInfo(): IDebugInfo {
```

Comment:
When I look at this, it seems like a new object can just be empty. I don't see a need for having the devServer there initially. The code when settings the devServer info can handle adding it if it is not there. 

I'm not even sure that you need this createDebugInfo() object. If the function just returns `{ }` then is it that useful?

Reply:
I like the design of initializing the object with all the fields/properties once and updating the values as it needs to.  That way if we add new fields/properties in the future we do it in just one place.